src/OpenTelemetry.Instrumentation.AWSLambda/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+* BREAKING: `ILambdaContext context` argument of all tracing methods of
+  `OpenTelemetry.Instrumentation.AWSLambda.AWSLambdaWrapper` was annotated as non-nullable.
+* Enabled null state analysis for `OpenTelemetry.Instrumentation.AWSLambda` and
+  `OpenTelemetry.Instrumentation.AWSLambda.Tests`. The interface will now
+  contain attributes for null-state static analysis. If null state analysis is
+  enabled your depending project, you may encounter new warnings.
+  ([#894](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/894))
```

Comment:
It should be a link to PR not a general issue.

```suggestion
  ([#1295](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/1295))
```

Reply:
Then it should be:
([#1295](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1295))

Reply:
suggestion fixed